### PR TITLE
Judith/oss 452 omsblog

### PR DESCRIPTION
**Pull request by judithpatudith**
## What is the urgency level of this change?
- [ ] Blocker
- [ ] High
- [x] Medium

## I have completed these items:
- [x] I have built locally and tested for broken links and formatting.
- [x] If appropriate, I have added redirects.

### CODE REVIEW

**Comment by judithpatudith** [comment]
@sascala when you get a chance, thank you!

File: src/blog/2016-11-22_oms_container_solution.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+category: universe
+description: How to monitor containers with the Operations Management Suite Container solution, on DC/OS.
+layout: article.jade
+collection: posts
+lunr: true
+---
+# Monitor containers on DC/OS with Microsoft's Operations Management Suite Container Solution
+
+Monitoring Docker containers can be challenging because containers can be created or destroyed at any time. The more containers you have, the more challenging monitoring becomes. Microsoft's [Operations Management Suite (OMS) Container Solution](https://docs.microsoft.com/en-us/azure/log-analytics/log-analytics-containers) helps users view their container inventory, performance, and logs, and to troubleshoot their containers. OMS Container Solution has been so well received that Microsoft is making its monitoring capabilities available for DC/OS clusters (regardless of the provider), as part of the Mesosphere Universe.
+
```
Comment (sascala) [inline src/blog/2016-11-22_oms_container_solution.md]: ^ well-received

File: src/blog/2016-11-22_oms_container_solution.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+OMS Container Solution is easy to install on DC/OS. Just choose the "msoms" package from the DC/OS web interface.
+
+<img src="/assets/images/blog/2016-12-08-image_2.png" alt="Packages in the DC/OS Universe" />
+
+When you install msoms, a containerized instance of OMS Agent for Linux gets installed on every private and public agent. This OMS instance scales with your DC/OS environment and sends monitoring data to the OMS Container Solutions web interface.
+
+## OMS Container Solution features
+
+* Centralize and correlate millions of logs from Docker containers at scale using journald
+* See real-time information about Container status, image, image tag, and affinity
```
Comment (sascala) [inline src/blog/2016-11-22_oms_container_solution.md]: "Container" should be lower case ^

**Comment by judithpatudith** [comment]
@sascala Edits made, thanks! Looks like its good to go!

**Comment by judithpatudith** [comment]
@sascala or @joel-hamill Please merge when you get a chance.

**Pull request merged**